recipe/meta.yaml

```diff
@@ -1,178 +1,334 @@
+        - wheel
       run:
-        - {{ pin_subpackage(name + '-core') }}
-        - mxnet
-        - pillow
+        - python
+        - numpy <1.29,>=1.21
+        - scipy <1.13,>=1.5.4
+        - pandas <2.2.0,>=2.0.0
+        - scikit-learn <1.5,>=1.3.0
+        - Pillow <11,>=10.0.1
+        - tqdm <5,>=4.38
+        - boto3 <2,>=1.10
+        - pytorch <2.1,>=2.0
+        - lightning <2.1,>=2.0.0
+        - requests <3,>=2.21
+        - jsonschema <4.18,>=4.14
+        - seqeval <1.3.0,>=1.2.2
+        - evaluate <0.5.0,>=0.4.0
+        - huggingface_accelerate <0.22.0,>=0.21.0
+
+        # transformers[sentencepiece]
+        - transformers <4.32.0,>=4.31.0
+        - sentencepiece !=0.1.92,>=0.1.91
+
+        - timm <0.10.0,>=0.9.5
+        - torchvision <0.16.0,>=0.14.0
+        - scikit-image <0.21.0,>=0.19.1
+        - text-unidecode <1.4,>=1.3
+        - torchmetrics <1.2.0,>=1.0.0
+        - nptyping <2.5.0,>=1.4.4
+        # Deviating from upstream. omegaconf-2.2.3 isn't fully compatible with Python 3.11.
+        - omegaconf <2.4.0,>=2.1.1
+        - {{ pin_subpackage(name + '.core', exact=True) }}
+        - {{ pin_subpackage(name + '.features', exact=True) }}
+        - {{ pin_subpackage(name + '.common', exact=True) }}
+        - pytorch-metric-learning <2.0,>=1.3.0
+        - nlpaug <1.2.0,>=1.1.10
+        - nltk <4.0.0,>=3.4.5
+        - openmim <0.4.0,>=0.3.7
+        - defusedxml <0.7.2,>=0.7.1
+        - jinja2 <3.2,>=3.0.3
+        - tensorboard <3,>=2.9
+        - pytesseract <0.3.11,>=0.3.9
+        # Exclusing because of imcompatibility with nvidia-ml-py.
+        # And anyway it's not imported by any autogluon code.
+        #- nvidia-ml 7.352.0
     test:
+      source_files:
+        - multimodal/tests
+      imports:
+        - autogluon.multimodal
       requires:
         - pip
         - pytest
-        - openml
+      commands:
+        # TODO: Figure out why patch isn't working
+        # - pip check
+        - pytest -v multimodal/tests -k 'not test_check_style' -m 'not slow'
+        - python -c 'import autogluon.multimodal.version; assert autogluon.multimodal.version.__version__ == "{{ version }}"'
+
+  - name: {{ name }}.tabular
+    build:
+      script: python -m pip install ./tabular -v --no-deps --no-build-isolation
+      script_env:
+        RELEASE=1
+    requirements:
+      host:
+        - python
+        - pip
+        - setuptools
+        - wheel
+      run:
+        - python
+        - numpy <1.29,>=1.21
+        - scipy <1.13,>=1.5.4
+        - pandas <2.2.0,>=2.0.0
+        - scikit-learn <1.5,>=1.3.0
+        - networkx <4,>=3.0
+        - {{ pin_subpackage(name + '.core', exact=True) }}
+        - {{ pin_subpackage(name + '.features', exact=True) }}
+        # Extra: all
+        - xgboost <2.1,>=1.6
+        - {{ pin_subpackage(name + '.core', exact=True) }}
+        - pytorch <2.1,>=2.0
+        - fastai <2.8,>=2.3.1
+        - lightgbm <4.2,>=3.3
+        - catboost <1.3,>=1.1
```

Comment:
Is ray-core missing? See https://github.com/autogluon/autogluon/blob/caccb0b816e5e428370650f935da4e793aa23e12/tabular/setup.py#L59

Reply:
This is not ray-core per say. The `ray` extra requires `autogluon.core[all]`. Our `autogluon.core` package already depends on ray.